cross/python3/Makefile

```diff
@@ -51,6 +55,8 @@ PATH := $(WORK_DIR)/../../../native/$(PKG_NAME)/work-native/install/usr/local/bi
 LD_LIBRARY_PATH := $(WORK_DIR)/../../../native/$(PKG_NAME)/work-native/install/usr/local/lib:$(LD_LIBRARY_PATH)
 export PATH LD_LIBRARY_PATH
 
+ENV += LIBFFI_INCLUDEDIR=$(STAGING_INSTALL_PREFIX)/include/
```

Comment:
I ensured by patching the configure mechanism of the python3 cross compiled package that the LIBFFI_INCLUDEDIR variable can actually be overridden explicitely using environment variable.
And here I ensure that the environment variable is appropriately defined.